src/components/Gallery.scss

```
@@ -0,0 +1,67 @@
+@use "../styles/_variables.scss" as *;
+
+.Gallery {
+  display: grid;
+  grid-gap: 0.625em;
+  grid-template-columns: repeat(auto-fit, minmax(12.5em, 1fr));
+  grid-auto-rows: 12.5em;
+  grid-auto-flow: dense;
+  padding: 0.625em;
+  height: 100vh;
+  overflow: scroll;
+
+  &__ImageContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__ImageCard {
+    max-height: 100%;
+    width: auto;
+    vertical-align: middle;
+    display: inline-block;
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    object-position: center;
+    border-radius: 0.625em;
+    border: solid 0.312em $color-primary-darkest;
+  }
+
+  &__ImageCard:hover {
+    border: solid 0.312em $button-dashboard-hover;
+  }
```

You can nest this inside the above:
```suggestion

    &:hover {
      border: solid 0.312em $button-dashboard-hover;
    }
  }
```